confgenerator/confgenerator.go

```diff
@@ -388,7 +388,7 @@ func generateFluentBitInputs(receivers map[string]*LoggingReceiver, pipelines ma
 	}
 	for _, pID := range sortedKeys(pipelines) {
 		p := pipelines[pID]
-		for _, rID := range p.Receivers {
+		for _, rID := range p.ReceiverIDs {
```

Comment:
Changed this for consistency because we were doing `XXXs` for logging and `XXXIDs` for metrics.

Reply:
Ack'd